server/coordinator.go

```diff
@@ -24,11 +24,12 @@ import (
 )
 
 const (
-	historiesCacheSize  = 1000
-	eventsCacheSize     = 1000
-	maxScheduleRetries  = 10
-	maxScheduleInterval = time.Minute
-	minScheduleInterval = time.Millisecond * 10
+	historiesCacheSize     = 1000
+	eventsCacheSize        = 1000
+	maxScheduleRetries     = 10
+	maxScheduleInterval    = time.Minute
+	minScheduleInterval    = time.Millisecond * 10
+	scheduleIntervalFactor = 1.3
```

Comment:
It's arbitrarily selected. We need a slower grow speed here.
If factor is 2, interval reaches max value 1min after about 13 retries, which takes less than 1.5min minutes in total.
When we can't schedule an operator in 1.5 minutes, it's not always true that the cluster is balanced, it may be caused by slow heartbeat or slow snapshot.